moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py

```diff
@@ -61,7 +61,7 @@
   class InvalidROSPkgException(Exception):
     pass
   def get_pkg_dir(pkg_name):
-    raise InvalidROSPkgException
+    raise InvalidROSPkgException('Failed to localize ROS package {}'.format(pkg_name))
```

Comment:
```suggestion
    raise InvalidROSPkgException('Failed to locate ROS package {}'.format(pkg_name))
```

`localize` would be the process of translating the package to another language.